### PR TITLE
Python SDK Overhaul

### PR DESCRIPTION
**Pull request by mrinehart-r7**
Changes:
- Rename Dockerfiles (this will, by effect, rename out plugin SDK images)
- Bulk build/tag/deploy same-version Dockerfiles
-- e.g. Python 2 full and slim images will get deployed in the same run
- Python 3.6 -> 3.7 (https://docs.python.org/3/whatsnew/3.7.html#porting-to-python-37)
-- Python 3.6 will remain in an older version
- Debian 8 -> 9 (https://www.debian.org/releases/stretch/amd64/release-notes/ch-whats-new.en.html#newdistro)
-- Debian 8 will remain in older SDK versions
- @jonschipp added a debug-slim image

### CODE REVIEW

**Comment by jmcadams-r7** [comment]
Oh I'll approve that all day. Looks like the CI died though.

**Comment by jschipp-r7** [comment]
@mrinehart-r7 Is this backward-compatible? Should we deploy a 3.7 image instead?

**Comment by mrinehart-r7** [comment]
@jschipp-r7 Going through all the possible regressions right now, so far the only thing that may impact us is the usage of `re.sub(...)` in a few of our plugins. Everything else should be fine based on this document https://docs.python.org/3/whatsnew/3.7.html#porting-to-python-37

File: .travis.yml [hunk 1]
```diff
@@ -16,8 +15,8 @@ env:
 install:
   - pip install tox-travis
 script:
-  - make test
-  - make tag
+  - bash -c "make test"
```
Comment (jschipp-r7) [inline .travis.yml]: Might make a comment here about why we're calling bash, because at first glance I'm like WTF

File: Makefile [hunk 2]
```diff
@@ -16,16 +18,20 @@ DOCKERFILE=$(shell echo $(TRAVIS_PYTHON_VERSION) | cut -d"." -f1)
...
-	docker build -t komand/python-3-plugin:test -f dockerfiles/3 .
+	# Build all 2/3-slim Docker images
+	@for dockerfile in $(wildcard dockerfiles/${DOCKERFILE}*); do \
+		F=$${dockerfile//dockerfiles\/}; \
+		docker build -t komand/python-$${F}-plugin -f dockerfiles/$${F} .; \
+	done
 
-pypy-3-image:
-	docker build -t komand/python-pypy3-plugin:test -f dockerfiles/pypy3 .
+#python-2-image:
```
Comment (jschipp-r7) [inline Makefile]: Remove all these comments?

**Pull request merged**